site/website/sidebars.json

```diff
@@ -71,6 +71,9 @@
       "6_contributing/1_developer_guide",
       "6_contributing/2_reporting_bugs",
       "6_contributing/3_credits"
+    ],
+    "Upgrade CRDs Version": [
```

Comment:
Should the sidebar be called Upgrade Guides with just the one section on `v0.7.x to v0.8.0`? I suspect there could be more in the future.